### PR TITLE
Remove bool return from runWeavelet

### PR DESCRIPTION
**Pull request by dnephin**
While browsing the code I noticed that the `bool` return from `Envelope.runWeavelet` was always `false`. Removing the return value simplifies the caller slightly. I'm opening this PR to see if you are interested in this small simplification. Thanks for considering it!

I often try to avoid `(bool, error)` returns, and opt to use a sentinel error value when possible. That may be an option in the future if `runWeavelet` needs to indicate a speicific error case.

### CODE REVIEW

File: runtime/envelope/envelope.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ func (e *Envelope) RunProfiling(_ context.Context, req *protos.RunProfiling) (*p
 // runWeavelet with the provided environment and wait for it to terminate.
 //
 // It returns true,_ if some incoming argument is bad any retries should be stopped.
```
Comment (mwhittaker) [inline runtime/envelope/envelope.go:265]: Delete this line now that it's obsolete?

**Review by mwhittaker — APPROVED** [review]
Nice find! Much better now! :shipit:

File: runtime/envelope/envelope.go (hunk 1 above)
Comment (dnephin) [reply runtime/envelope/envelope.go]: Ah, yes! I've pushed another commit that removes this line

**Comment by mwhittaker** [comment]
GitHub says this branch is out-of-date with the base branch. Mind rebasing and squashing on top of main?

**Comment by dnephin** [comment]
Rebase and squash done!

**Pull request merged**